Public/hulu.html

```diff
@@ -174,10 +174,10 @@ <h4>Hulu + Live TV</h4>
         <div class="divider"></div>
 
         <div class="social-icons">
-          <a href="#"><img src="img/facebook.svg" alt="" /></a>
-          <a href="#"><img src="img/twitter.svg" alt="" /></a>
-          <a href="#"><img src="img/youtube.svg" alt="" /></a>
-          <a href="#"><img src="img/instagram.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-facebook.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-twitter.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-youtube.svg" alt="" /></a>
+          <a href="#"><img src="../assets/Images/hulu-instagram.svg" alt="" /></a>
```

Comment:
Like this one,  **assets/Images/hulucloneimg/instagram.svg**   is the correct address according to your commits but you did it wrong. 
you changed the svg image name in html file but not the file name of original svg, also you didn't specify the folder.